ts-tests/__tests__/blocks.test.ts

```diff
@@ -0,0 +1,48 @@
+import { MetaDContainer } from '../containers';
+
+const container = new MetaDContainer();
+
+beforeAll(async () => {
+  await container.start();
+});
+
+afterAll(async () => {
+  await container.stop();
+});
+
+it('should generate', async () => {
+  const b0 = await container.web3.eth.getBlockNumber();
+  expect(b0).toStrictEqual(0);
```

Comment:
Do the different metad containers that are started broadcast their state/transactions to each other? These block numbers are always 1 off then I run these tests as a whole (`npm test` from package root)

Reply:
should be fixed after attached the network https://github.com/DeFiCh/metachain/pull/33/commits/ca92644bc1c3079674dbd5ae4461e62e5cb0bde6

Reply:
Nice, passes on my end! Probably because each container(node) needs to be isolated in its own network 🤔